orbbec_camera/src/ros_service.cpp

```diff
@@ -1069,6 +1072,63 @@ void OBCameraNode::resetCaptureServiceVariables() {
   number_of_depth_frames_captured_ = 0;
 }
 
+void OBCameraNode::handleChangeStateRequest(
+    const std::shared_ptr<lifecycle_msgs::srv::ChangeState::Request> request,
+        std::shared_ptr<lifecycle_msgs::srv::ChangeState::Response> response) {
+    
+  RCLCPP_INFO(logger_, "Received request to change state '%d' with label '%s'",
+                     request->transition.id, request->transition.label.c_str());
+  
+  response->success = true;
+  if(request->transition.id == lifecycle_msgs::msg::Transition::TRANSITION_ACTIVATE) {
+    RCLCPP_INFO_STREAM(logger_, "Recieved request to turn ON camera streams");
+    if (pipeline_started_){
+      RCLCPP_WARN_STREAM(logger_, "Camera streams already ON");
+      response->success = false;
```

Comment:
probably a philosophical discussion but some may say this should be true... time will tell

Reply:
```suggestion
      response->success = true;
```
ok changed my mind, I think this *should* be true :D